pkgs/development/python-modules/pysmartapp/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, six
+, httpsig
+}:
+
+buildPythonPackage rec {
+  pname = "pysmartapp";
+  version = "0.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1i5f6as4f0jnba2zynni0v0y9fkf2qfavpyz73nk78as4j8ysnr9";
+  };
+
+  propagatedBuildInputs = [
+    six
```

Comment:
The [`install_requires`](https://github.com/andrewsayre/pysmartapp/blob/master/setup.py) do not mention `six`.